### DISCUSSION TITLE
Using Django dumpdata and loaddata

### DISCUSSION THREAD

**Discussion by JannikZed** [saleor/saleor#9003] [Q&A]
Hi there,

We would like to test a "simple" way, to work with different saleor instances in different environments - in most cases, we would love to find a way, to improve the developer experience that we got. We came across the `python manage.py dumpdata` that sounded like a good fit, to export for example product data and use it as seeding data to start a new saleor instance from scratch. 

Does anybody has some experience in using this kind of data export and import? We can't get it running easily. We got errors like `Problem installing fixture '/app/mountFolder/productsAttrbitues.json': Invalid model identifier: 'attribute.attribute'`

**Comment by IKarbowiak** [comment]
Hi @JannikZed,

Are you sure the data dump is from the same Saleor version as the version you are loading data into? Models may change between versions and then could be a problem with loading data, as the given model might be in a different direction or might have different fields.

**Comment by timuric** [comment]
@JannikZed have you considered using Saleor cloud, which offers sandbox management?
